app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -279,5 +279,5 @@ def update_trace_status
 
 class ::Host::Managed::Jail < Safemode::Jail
   allow :content_source, :subscription_manager_configuration_url, :rhsm_organization_label,
-        :host_collections, :comment
+        :host_collections, :comment, :pools, :hypervisor_host
```

Comment:
To fix the `Jail` errors we're seeing, looks like you should add `lifecycle_environment, :content_view` here